devices/eurotronic/eurotronic_spzb0001_trv.json

```diff
@@ -0,0 +1,373 @@
+        {
+          "name": "config/mode",
+          "refresh.interval": 3660,
+          "read": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "write": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "dt": "0x22",
+            "ep": 1,
+            "eval": "if (Item.val == 'off') { R.item('config/hostflags').val & ~0x04 | 0x20 } else if (Item.val == 'heat') { R.item('config/hostflags').val & ~0x20 | 0x04 } else if (Item.val == 'auto') { R.item('config/hostflags').val & ~0x20 & ~0x04 }",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "parse": {
+            "at": "0x4008",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "if ((Attr.val & 0x30) !== 0) { Item.val = 'off' } else if ((Attr.val & 0x04) !== 0) { Item.val = 'heat' } else { Item.val = 'auto' }",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "default": "auto"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/errorcode",
+          "refresh.interval": 3660,
+          "parse": {
+            "at": "0x4002",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "var res = ''; if (Attr.val & 0x08) { res = 'E1'; } else if (Attr.val & 0x10) { res = 'E2'; } else if (Attr.val & 0x20) { res = 'E3'; } else { res = 'none'; } Item.val = res;",
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "read": {
+            "at": "0x4002",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr",
+            "mf": "0x1037"
+          },
+          "default": "none"
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/on",
+          "refresh.interval": 660,
+          "read": {
+            "at": "0x0000",
+            "cl": "0x0006",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0006",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": false
+        },
+        {
+          "name": "state/temperature",
+          "refresh.interval": 660,
+          "read": {
+            "at": "0x0000",
+            "cl": "0x0201",
+            "ep": 1,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/valve",
+          "refresh.interval": 660,
+          "read": {
+            "fn": "none"
+          },
```

Comment:
Is there any particular reason not to read the attribute? I would've left the default read function here.